src/python/nxt_python_asgi.h

```diff
@@ -22,7 +22,21 @@ typedef struct {
     nxt_unit_request_info_t  *req;
     uint64_t                 content_length;
 } nxt_py_asgi_add_field_ctx_t;
-
+#ifdef NXT_HAVE_ASYNCIO_RUNNER
+typedef struct {
+    nxt_queue_t      drain_queue;
+    PyObject         *loop_run_until_complete;
+    PyObject         *loop_create_future;
+    PyObject         *loop_create_task;
+    PyObject         *loop_call_soon;
+    PyObject         *loop_add_reader;
+    PyObject         *loop_remove_reader;
+    PyObject         *quit_future;
+    PyObject         *quit_future_set_result;
+    PyObject         **target_lifespans;
+    PyObject         *runner;
+} nxt_py_asgi_ctx_data_t;
+#else
```

Comment:
Please just add the member to the structure, wrapped in preprocessor, rather than copying the entire structure.